cmd/protect.go

```diff
@@ -74,6 +75,23 @@ func runProtect(cmd *cobra.Command, args []string) {
 		log.Fatal().Err(err).Msg("")
 	}
 
+	gitleaksPath, err:= cmd.Flags().GetString("gitleaks-ignore-path")
```

Comment:
@pacorreia could we rename this variable to `gitleaksIgnorePath`?

Reply:
Done